src/main/java/io/temporal/internal/sync/WorkflowThreadImpl.java

```diff
@@ -55,7 +55,7 @@
   class RunnableWrapper implements Runnable {
 
     private final WorkflowThreadContext threadContext;
-    private final DecisionContext decisionContext;
+    private final ReplayWorkflowContext replayWorkflowContext;
```

Comment:
why ReplyWorkflowContext?  Looks like a comment would be helpful here.

Reply:
Added TODO to remove from there.